npm-shrinkwrap.json

```diff
@@ -806,6 +806,7 @@
 			"version": "7.2.0",
 			"resolved": "https://registry.npmjs.org/@babel/plugin-syntax-jsx/-/plugin-syntax-jsx-7.2.0.tgz",
 			"integrity": "sha512-VyN4QANJkRW6lDBmENzRszvZf3/4AXaj9YR7GwrWeeN9tEBPuXbmDYVU9bYBN0D70zCWVwUy0HWq2553VCb6Hw==",
+			"dev": true,
```

Comment:
Interesting 🤔

```
wp-calypso@0.17.0
└─┬ @automattic/calypso-build@2.0.0 -> …/packages/calypso-build
  ├─┬ @babel/plugin-transform-react-jsx@7.3.0
  │ └── @babel/plugin-syntax-jsx@7.2.0
  └─┬ @babel/preset-react@7.0.0
    ├─┬ @babel/plugin-transform-react-jsx-self@7.2.0
    │ └── @babel/plugin-syntax-jsx@7.2.0  deduped
    └─┬ @babel/plugin-transform-react-jsx-source@7.2.0
      └── @babel/plugin-syntax-jsx@7.2.0  deduped
```

Reply:
The root `package.json` used to declare the plugin as a normal dependency. Now it's a dev dependency by virtue of `calypso-build`.